dace/frontend/python/replacements.py

```diff
@@ -2039,6 +2149,11 @@ def _set_tasklet_params(ufunc_impl: Dict[str, Any],
     # Remove input connectors related to constants
     # and fix constants/symbols in the tasklet code
     for i, arg in reversed(list(enumerate(inputs))):
+        inp_conn = inp_connectors[i]
+        if casting and casting[i]:
+            repl = "{c}({o})".format(c=str(casting[i]).replace('::', '.'),
```

Comment:
ugh... maybe we can add a `FIXME` comment here for later? (so that we have better functions for this)

Reply:
I believe this is fine. However, I would find it interesting if I changed the connector type, and instead of validation throwing an error, the backend implemented the casting. This would open the way to have casting work with vectorization.